Revit_Core_Engine/Query/OrientationAngle.cs

```diff
@@ -58,31 +57,38 @@ public static double OrientationAngle(this FamilyInstance familyInstance, RevitS
         [Description("Get the orientation angle of a duct.")]
         [Input("Autodesk.Revit.DB.Mechanical.Duct", "Revit duct.")]
         [Input("BH.oM.Adapters.Revit.Settings.RevitSettings", "Revit settings.")]
-        [Output("double", "Orientation angle of a duct.")]
+        [Output("double", "Orientation angle of a duct in radians.")]
```

Comment:
Variable names are wrong, descriptions could be improved too.

Reply:
Corrected